src/scss/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
```

Comment:
I would recommend importing original bootstrap functions, so you can fully unluck the potential of customizing bootstrap with scss. This would go on top over the import of customized variables in example, line 1

`@import "../../node_modules/bootstrap/scss/functions";`